packages/itwin/reports-config-widget/api/reports-config-widget-react.api.md

```diff
@@ -0,0 +1,129 @@
+## API Report File for "@itwin/reports-config-widget-react"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+/// <reference types="react" />
+
+import type { AccessToken } from '@itwin/core-bentley';
+import { ExtractionClient } from '@itwin/insights-client';
+import { IModelsClient } from '@itwin/imodels-client-management';
+import type { Localization } from '@itwin/core-common';
+import { MappingsClient } from '@itwin/insights-client';
+import { default as React_2 } from 'react';
+import type { Report } from '@itwin/insights-client';
+import { ReportsClient } from '@itwin/insights-client';
+import { StagePanelLocation } from '@itwin/appui-react';
+import { StagePanelSection } from '@itwin/appui-react';
+import type { UiItemsProvider } from '@itwin/appui-react';
+import type { Widget } from '@itwin/appui-react';
+
+// @public (undocumented)
+export class BulkExtractor {
```

Comment:
I wonder if this should be exported. It's a pretty hacky solution for tracking running extractions.

Reply:
It's out in the wild already. I can just mark it internal.

Reply:
Ouch. 
![image](https://github.com/iTwin/viewer-components-react/assets/26608199/a0333962-9de6-4ab8-a412-df5e4c0c84c0)
I believe the reason it's there is to allow mocking of the BulkExtractor.

Reply:
Sounds like a bigger change than I expected initially. I think it's better to defer fixing this for now.